docs/example/saxml/README.md

```diff
@@ -0,0 +1,94 @@
+# Deploy SaxML Multihost with LWS on GKE
+
```

Comment:
could you add:
In the example, we will use LeaderWorkerSet to deploy a multi-host inference instance on TPUs with Saxml. You could follow the steps here to setup your TPU clusters, node pools, gcs bucket, workload identity etc and use LeaderWorkerSet to deploy the model servers.

Reply:
Done